SignalShareExtension/ShareViewController.swift

```diff
@@ -12,6 +12,12 @@ import PromiseKit
 @objc
 public class ShareViewController: UINavigationController, ShareViewDelegate, SAEFailedViewDelegate {
 
+    enum ShareViewControllerError: Error {
+        case assertionError(description: String)
+        case unsupportedMedia
+        case NotRegistered(description: String)
```

Comment:
style nit, enum should be lowercased. 

Does this need a description? Seems to speak for itself.

Reply:
Done.